ui.R

```diff
@@ -69,9 +69,7 @@ ui <- function(input, output, session) {
     cookies_banner_ui(
       name = "Children's Social Care - Outcomes and Enablers"
     ),
-    dfeshiny::custom_disconnect_message(
-      links = c(site_primary, site_overflow)
-    ),
+    customDisconnectMessage(),
```

Comment:
Removing dfeshiny:: from here will be the reason for the error seen below: 
![image](https://github.com/user-attachments/assets/c1c8acf9-25bd-4719-a3f3-f76a7c7f0ce5)

Reply:
@yauemily I'm wrong actually - what was the reason for changing this? dfeshiny documentation shows that the old way was right?? https://dfe-analytical-services.github.io/dfeshiny/reference/custom_disconnect_message.html

Reply:
@chfoster I have no idea. I think it was Rich, who changed it